framework/wazuh/rbac/orm.py

```diff
@@ -2402,6 +2556,46 @@ def replace_role_policy(self, role_id: int, current_policy_id: int, new_policy_i
 
         return False
 
+    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
+                     to_id: Optional[int] = None):
```

Comment:
Please add the docstring

```suggestion
    def migrate_data(self, manager, source: str, target: str, from_id: Optional[int] = None,
                     to_id: Optional[int] = None) -> None:
```

Reply:
Fixed in 402c4224dd39f3a6dd3cefca2e48f78cd7c872a2.